src/plugins/mdraid.c

```diff
@@ -1061,7 +1061,7 @@ gchar* bd_md_get_md_uuid (const gchar *uuid, GError **error) {
  * @name: name of the MD RAID
  * @error: (out): place to store error (if any)
  *
- * Returns: path to the @name MD RAID's device node or %NULL in case of error
+ * Returns: @name MD RAID's device node or %NULL in case of error
```

Comment:
"*device node of the @name MD RAID or %NULL in case of error*" would probably sound better here.